relay-single-test.ts

```diff
@@ -117,31 +117,33 @@ export const newSub_multiple_filters = (url: string) => async () => {
         kind: NostrKind.Long_Form,
         content: "test2",
     });
-    try {
-        const stream = await relay.newSub(
-            "multiple filters",
-            {
-                ids: [event_1.id],
-                limit: 1,
-            },
-            {
-                authors: [event_2.pubkey],
-                limit: 1,
-            },
-        );
-        if (stream instanceof Error) fail(stream.message);
+    {
+        const err1 = await relay.sendEvent(event_1);
+        if (err1 instanceof Error) fail(err1.message);
+        const err2 = await relay.sendEvent(event_2);
+        if (err2 instanceof Error) fail(err2.message);
```

Comment:
always send events before reading unless we are testing the behavior of absence of events. Not all relays have enough events that's always ready to be read